src/cli/cli.ts

```diff
@@ -51,42 +51,42 @@ async function main() {
       progressBar.itemDone(name);
     }
   });
-  term.bold('\n\nAPIs\n\n');
-  term.table(
-    reportData.map((report) => [
-      report.status === 'passed' && report.run.meetsSLA ? '^G√ ' : '',
-      `${report.status === 'passed' && report.run.meetsSLA ? '' : '^R'}${
-        report.query.signature || report.query.query
-      } ${
-        report.status === 'passed' && !program.verbose
-          ? ''
-          : `${report.errors.length ? '\n\n' + report.errors[0] + '\n' : ''}`
-      }\n`,
-      `${report.run.meetsSLA ? '^G' : '^R'}${report.run.ms}ms `,
-    ]),
-    {
-      hasBorder: true,
-      borderChars: 'lightRounded',
-      borderAttr: { color: 'blue' },
-      contentHasMarkup: true,
-      textAttr: { bgColor: 'default' },
-      width: 80,
-      fit: true,
-    }
-  );
+
+  if (program.verbose) {
```

Comment:
Looking at the documentation for the verbose flag it appears that it was originally designed for outputting the full query information rather than just the signature.   This allows copy and pasting of a query to test out why it's failing.   I think it would be nice to keep that feature and add a new flag to control the hiding of the table.